lib/matplotlib/font_manager.py

```diff
@@ -184,7 +184,7 @@ def win32FontDirectory():
     Return the user-specified font directory for Win32.  This is
     looked up from the registry key::
 
-      \\HKEY_CURRENT_USER\Software\Microsoft\Windows\CurrentVersion\Explorer\Shell Folders\Fonts
+      \\HKEY_CURRENT_USER\\Software\\Microsoft\\Windows\\CurrentVersion\\Explorer\\Shell Folders\\Fonts
```

Comment:
The first one is supposed to be doubled, so it should be _4_ when escaped, but maybe it'd be simpler to make it a raw string instead?